pybossa/sched.py

```diff
@@ -237,9 +239,7 @@ def locked_scheduler(query_factory):
     def template_get_locked_task(project_id, user_id=None, user_ip=None,
```

Comment:
Are the following properties used any longer or can they be removed from this method's parameter list? Or is this for backwards compatibility with the calling method?

```
user_ip
external_uid
orderby
desc
```

Reply:
Yes, those are needed as they are passed by the caller.